### DISCUSSION TITLE
Notes - 12th April 2024

### DISCUSSION THREAD

**Discussion by mmcky** [QuantEcon/reading-group-2024#25] [General]
## Heavy Tailed Distributions

- check into `yf.download` and put it in its own cell block and `hide-output`
- update figures and images to be numbered
- overlay a `normal` distribution on top of the `returns` picture before `19.1.4`
- should we use `rank-size` plot of `qq` plot? to show CCDF
- see if there is a statistical test for power laws and perhaps write a `seealso` resource. Can you test for a `pareto` distribution. 

**Comments:**

- How does equation `19.4` work? 

## Racial Segregation

- @HumphreyYang to load @Jiarui-ZH notes here

- `Jump chain title` should be deleted
- The exercise solution needs to include the discussion and text at the end of the lecture (move `solution-end`)
- pseudocode could be wrapped in a special admonition
- fix the `Entering Loop String` repetition in the code output. Put the figures side by side for better comparisons of the neighbourhood formation. 

- We could focus the lecture a bit more clearly on simple `local` interactions and how they lead to unexpected macro outcomes
- Add some small network diagrams in `preferences` section to diagramatically show `happines`
- add a `happiness` distribution as the iterations go from 0 to 7 to see how that distriubtion changes. (What proportion of people are happy in Cycle 0?)

**Comment by HumphreyYang** [comment]
Many thanks @mmcky,

This is the note I am presenting from @Jiarui-ZH today:
[Suggestions to Racial Segregation lecture.docx](https://github.com/QuantEcon/reading-group-2024/files/14952161/Suggestions.to.Racial.Segregation.lecture.docx)

**Comment by HumphreyYang** [comment]
The assignment next week will be:

@pgrosser1: https://intro.quantecon.org/solow.html
@Jiarui-ZH: https://intro.quantecon.org/scalar_dynam.html
@mbek0605: https://intro.quantecon.org/cobweb.html

(CC @jstac)

**Comment by HumphreyYang** [comment]
The issues discussed above are now added to the issue trackers in 

https://github.com/QuantEcon/lecture-python-intro/issues/418
https://github.com/QuantEcon/lecture-python-intro/issues/419
